TrackletAlgorithm/TrackletProcessor.h

```diff
@@ -431,51 +359,21 @@ ::rinv * const rinv, TrackletParameters::PHI0PAR * const phi0, TC::Types::z0 * c
 
 // This function calls calculate_DXDY, defined in
 // TrackletCalculator_calculate_DXDY.h, and applies cuts to the results.
-#include "TrackletCalculator_calculate_LXD1.h"
+//#include "TrackletCalculator_calculate_LXD1.h"
 
-#include "TrackletCalculator_calculate_DXDY.h"
+//#include "TrackletCalculator_calculate_DXDY.h"
```

Comment:
@aryd Since this file has been removed, let's just remove this line:
```suggestion
```

Reply:
Fixed

Reply:
Fixed